.travis.yml

```diff
@@ -10,7 +10,6 @@ branches:
   except:
     - /^experimental\/.*$/
     - /^exp\/.*$/
-    - /^development\/.*$/
     - /^dev\/.*$/
```

Comment:
Test and build "development" branches. Development is a branch suffix that now can be used that will push and build a provisioner-image so testing with new provisioner-image is possible